Orange/widgets/unsupervised/owmds.py

```diff
@@ -157,19 +156,14 @@ def __init__(self):
         #: Input data table
         self.signal_data = None
 
-        self._pen_data = None
-        self._brush_data = None
-        self._shape_data = None
-        self._size_data = None
-        self._label_data = None
         self._similar_pairs = None
-        self._scatter_item = None
-        self._legend_item = None
-        self._selection_mask = None
-        self._subset_mask = None  # type: Optional[numpy.ndarray]
+        self._subset_mask = []  # type: Optional[np.ndarray]
```

Comment:
This used to be `self._subset_mask = None`. Why change it to `[]`? If this is to avoid having to test for `None` in [line 649](https://github.com/biolab/orange3/pull/2448/files#diff-33a6f21b7026aae1b62fbdfca1d53b86R649), you shouldn't set it to `None` in line 341. It is set to `np.ndarray` in 575.

The type hint, `# type: Optional[np.ndarray]`, suggests that this can be `None` or `ndarray`. Not an (empty) list.

This is a mess. Please revert it to how it was before: `None` and `ndarray`, not a list.

Reply:
Fixed.